kernel/term_typing.ml

```diff
@@ -93,10 +93,36 @@ let infer_declaration (type a) ~(trust : a trust) env (dcl : a constant_entry) =
         cook_context = ctx;
       }
 
+    (** Primitives cannot be universe polymorphic *)
+    | PrimitiveEntry ((t,uctx),op_t) ->
+      let env = push_context_set ~strict:true uctx env in
+      let j = infer env t in
+      check_primitive_type env op_t t;
```

Comment:
Why do we put the type in the entry when the kernel has to know what it is regardless?